src/components/shared/misc/SummarisedMultiSelector.vue

```diff
@@ -0,0 +1,247 @@
+    <div class="dropdown-wrapper">
+        <semantic-form-dropdown
+            ref="summarisedSelector"
+            v-show="!readOnly"
+            v-model="model"
+            :options="options"
+            :data-tooltip="summary"
+            :placeholder="placeholder"
+            :data-inverted="true"
+            :auto-update="false"
+            :autoUpdateOptions="false"
+            :full_text_search="true"
+            :title_field="fields.title"
+            :value_field="fields.value"
+            :inline="inline"
+            :fluid="fluid"
+            search multiple
+            :settings="dropdownSettings"
+            @dropdown-selected="dropdownSelected"
+        />
+
+        <div v-if="readOnly && selectedLength < 1" :data-inverted="true">
+            {{ placeholder }}
+        </div>
+
+        <div v-else-if="readOnly && selectedLength < 2" :data-tooltip="selectedItems" :data-inverted="true">
+            {{ selectedItems }}
+        </div>
+
+        <div v-else-if="readOnly && selectedLength > 1" :data-tooltip="selectedItems" :data-inverted="true">
+             {{ selectedLength }} Selected
+        </div>
+    </div>
+</template>
+
+<script>
+    /**
+     * A custom multi-select dropdown useful for multi selection scenarios
+     *
+     * @example ./croud-summarised-mutli-selector.md
+     */
+
+    export default {
+
+        name: 'croud-summarised-multi-selector',
+
+        props: {
+            /**
+            * Pass an array of options to display in the dropdown
+            */
+            options: {
+                type: Array,
+                default() {
+                    return []
+                },
+            },
+
+            /**
+            * Maps the dropdown to the title/value fields on your options array.
+            */
+            fields: {
+                type: Object,
+                default() {
+                    return {
+                        title: 'name',
+                        value: 'id',
+                    }
+                },
+            },
+
+            /**
+            * The current value of the dropdown, v-model alias
+            */
+            value: {
+                required: true,
+                type: [String, Array],
+                default() {
+                    return ''
+                },
+            },
+
+            /**
+            * An optional placeholder for the dropdown
+            */
+            placeholder: {
+                type: String,
+                default() {
+                    return ''
+                },
+            },
+
+            /**
+            * Default text for the dropdown summary popup
+            */
+            defaultSummary: {
+                type: String,
+                default() {
+                    return 'Select an item'
+                },
+            },
+
+            /**
+            * Which field in the options to use in the summary popup
+            */
+            summaryField: {
```

Comment:
Could the summary field be part of the `fields` prop?

Reply:
It could, the only reason i didn't is so we didn't need to define all of the fields but i guess we could utilise defaultsDeep to remedy that. cheers